dashboards-reports/public/components/report_definitions/report_settings/report_settings_helpers.tsx

```diff
@@ -196,3 +196,8 @@ export const handleDataToVisualReportSourceChange = (
   delete reportDefinitionRequest.report_params.core_params.excel;
   reportDefinitionRequest.report_params.core_params.report_format = 'pdf';
 };
+
+export const getReportSourceFromContextMenu = (url: string) => {
```

Comment:
Np: I feel it's better to rename this function to be `getSomethingFromUrl`

Reply:
Sure, can change. that's more descriptive